libass/ass_blur.c

```diff
@@ -59,9 +58,7 @@ inline static const int16_t *get_line(const int16_t *ptr, uintptr_t offs, uintpt
 
 inline static void copy_line(int16_t *buf, const int16_t *ptr, uintptr_t offs, uintptr_t size)
 {
-    ptr = get_line(ptr, offs, size);
-    for (int k = 0; k < STRIPE_WIDTH; ++k)
-        buf[k] = ptr[k];
+    memcpy(buf, get_line(ptr, offs, size), STRIPE_WIDTH * sizeof(buf[0]));
```

Comment:
Similarly, this can be split out.

Reply:
I think that quite falls under "Simplify blur algorithm". If I split that into another commit then assembly modifications of the old functions (shrink/expand) should be separated too.